test/Table_Tests/src/Database/Upload_Spec.enso

```diff
@@ -2,12 +2,14 @@ from Standard.Base import all
 import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
 
 from Standard.Table import all
-from Standard.Table.Errors import Missing_Input_Columns
+from Standard.Table.Errors import all
 
 from Standard.Database import all
 from Standard.Database.Errors import all
 from Standard.Database.Internal.Result_Set import result_set_to_table
-from Standard.Database.Extensions.Upload_Default_Helpers import get_primary_key
+from Standard.Database.Extensions.Upload_Default_Helpers import get_primary_key, default_key_columns
+import Standard.Database.Data.Column_Constraint.Column_Constraint
+import Standard.Database.Data.Column_Description.Column_Description
```

Comment:
we should add these to `Main.enso` export. Users will need to make them.

Reply:
Fair! I will add `Column_Description`, but do you think I should add `Column_Constraint` too?

My thought was that it is a more advanced feature, so I'd keep it 'hidden'. Technically, the IDE should add an import once it _is_ used.

Reply:
I guess and probably little reason to build outside but at least currently we export all the API types we use.

Reply:
Possibly in Database, but in Base we have Ref/State that are not exported at the toplevel. IMO the constraints could have a similar role here.